package.json

```diff
@@ -56,6 +56,7 @@
   "dependencies": {
     "babel-runtime": "^6.6.1",
     "duplex-passthrough": "github:diasdavid/duplex-passthrough",
+    "eventemitter2": "^1.0.3",
```

Comment:
I've been using this one for a while(years) now. Before this was the one that ran 100% on the browser. But I can with all the transpilation nowadays, we can use the default one.